src/docs/devices/M5Stack-Fire/index.md

```diff
@@ -0,0 +1,212 @@
+---
+title: M5Stack Fire
+date-published: 2023-10-18
+type: misc
+standard: global
+board: esp32
+project-url: https://docs.m5stack.com/en/core/fire
+---
+
+## Product Images
+
+![M5Stack Fire](M5StackFire.webp "M5Stack Fire")
+
+## Description
+
+M5Stack FIRE Kit features an ESP32 Core, 16M Flash + 8M PSRAM, 9-Axis IMU sensor (6-Axis posture acceleration measurement + 3-Axis magnetic measurement), power management IC, speaker, microphone, and a 2 inch 320x240 LCD screen. The middle part is called M5GO base which provides a 500 mAh battery, M-BUS socket, LED bar and three more GROVE Ports. The bottom part is a charge table, which can be connected to the M5GO base via POGO pins.
+
+## GPIO Pinout
+
+|   Pin  | Function    |
+|:------:|-------------|
+| GPIO39 |   Button A  |
+| GPIO38 |   Button B  |
+| GPIO37 |   Button C  |
+| GPIO34 | Microphone Pin |
+| GPIO25 | Speaker Pin 1 |
+| GPIO26 | Speaker Pin 2 |
+| GPIO22 | GROVE A SCL |
+| GPIO21 | GROVE A SDA |
+| GPIO36 | GROVE B SCL |
+| GPIO26 | GROVE B SDA |
+| GPIO16 | GROVE C RXD |
+| GPIO17 | GROVE C TXD |
+| GPIO15 | SIG Pin     |
+
+### MBus Pin Map
+
+![Pin Map](core1_mbus.webp "M5Stack Pin Map")
+
+## MPU6886 and BMM150
+
+![MPU6886 BMM150](MPU6886_BMM150.webp "MPU6886 BMM150")
+
+| I2C Address | Function                                |
+|-------------|-----------------------------------------|
+| 0x68        | MPU6886 6-Axis MotionTracking Sensor |
+| 0x10        | BMM150 3-Axis Geomagnetic Sensor     |
+
+## IP5306 Power Management
+
+M5Stack Fire uses a customized version of the IP5306 Power Management circuit
+[Customized IP5306 Datasheet]("https://github.com/m5stack/M5-Schematic/blob/master/Core/IIC_IP5306_REG_V1.4.pdf")
+
+| I2C Address | Function                                |
+|-------------|-----------------------------------------|
+| 0x75        | IP5306 Power Management |
+
+### IP5306 Example Configuration
+
+```yml
+external_components:
+  - source:
+      type: git
+      url: https://github.com/ssieb/custom_components
+    components: [ ip5306 ]
+
+ip5306:
+  battery_level:
+    name: Battery Level
+  charger_connected:
+    id: connected
+  charge_full:
+    id: full
+```
+
+Using the [IP5306 ESPHome Component]("https://github.com/ssieb/custom_components/tree/master/components/ip5306") by [@ssieb]("https://github.com/ssieb")
+
+## Example Configuration
```

Comment:
~~~suggestion
## Wifi Issues due to GPIO15

When the M5GO bottom is attached, GPIO15 (side lights pin) needs to be setup as an output pin, otherwise Wifi will fail to connect frequently. If you do not want to setup the side lights component (to save memory, for example), you can use the following alternative instead:

```yml
output:
  - platform: gpio
    id: wifi_fix_output
    pin: GPIO15
```

Source: [Official M5Stack repo](https://github.com/m5stack/M5Unified/blob/d26d2415f409b664631ff066ceeaffd6a753a4ff/src/M5Unified.cpp#L499).

## Example Configuration
~~~